src/external_cli/cargo/mod.rs

```diff
@@ -0,0 +1,117 @@
+#![expect(dead_code, reason = "Will be used for `bevy build` and `bevy run`")]
+use clap::{ArgAction, Args};
+
+use super::arg_builder::ArgBuilder;
+
+pub(crate) mod build;
+
+// TODO: Support customizing the executable via an env var
+// https://github.com/TheBevyFlock/bevy_cli/issues/71
+const PROGRAM: &str = "cargo";
```

Comment:
(Non-blocking) If this is turned into a function, we can pretty easily implement #71:

```suggestion
fn program() -> OsString {
   env::var_os("BEVY_CLI_CARGO").unwrap_or("cargo".into())
}
```

Reply:
Ok sure, we can do it in this PR :)